src/modules/flight_mode_manager/tasks/Utility/StickAccelerationXY.cpp

```diff
@@ -57,13 +57,24 @@ void StickAccelerationXY::resetPosition(const matrix::Vector2f &position)
 
 void StickAccelerationXY::resetVelocity(const matrix::Vector2f &velocity)
 {
-	_velocity_setpoint = velocity;
+	if (velocity.isAllFinite()) {
+		_velocity_setpoint = velocity;
+
+	} else {
+		_velocity_setpoint.zero();
+	}
 }
 
 void StickAccelerationXY::resetAcceleration(const matrix::Vector2f &acceleration)
 {
-	_acceleration_slew_rate_x.setForcedValue(acceleration(0));
-	_acceleration_slew_rate_y.setForcedValue(acceleration(1));
+	if (acceleration.isAllFinite()) {
+		_acceleration_slew_rate_x.setForcedValue(acceleration(0));
+		_acceleration_slew_rate_y.setForcedValue(acceleration(1));
+
+	} else {
+		_acceleration_slew_rate_x.setForcedValue(0.f);
+		_acceleration_slew_rate_y.setForcedValue(0.f);
```

Comment:
I'd rather leave the slew rate where it is instead of jumping to zero. This could avoid a strong twitch.